packages/shared/src/config/modules.ts

```diff
@@ -78,6 +78,16 @@ export const choiceCardsBannerYellow: ModuleInfo = getDefaultModuleInfo(
     'banners/choiceCardsBanner/ChoiceCardsBannerYellow',
 );
 
+export const choiceCardsButtonsBannerBlue: ModuleInfo = getDefaultModuleInfo(
```

Comment:
Think you to add corresponding entries for this and next at line 136 (export const moduleInfo)?

Reply:
Yes will update!